src/views/include/inputs/github-handles.html

```diff
@@ -0,0 +1,14 @@
+{{ govukTextarea({
+  errorMessage: errors.description if errors,
```

Comment:
Errors for description box doesn't render when you submit the form, need to update to:

`errorMessage: errors.github_handles if errors`